CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Enhance contribution guidelines: Refactored shell scripts, added additional steps. (#91)
 
+### Changed
+- Updated the error display to a modern UI.
```

Comment:
Please add the PR number in parenthesis at the end of the line. And we probably do not need two entries in the CHANGELOG, I would say it is the same work item.